app/src/main/java/com/orange/ouds/app/ui/MainScreen.kt

```diff
@@ -86,7 +86,7 @@ fun MainScreen(
     var changeThemeSettingsDialogVisible by rememberSaveable { mutableStateOf(false) }
 
     OudsTheme(
-        themeContract = mainState.themeState.currentTheme,
+        theme = mainState.themeState.currentTheme,
```

Comment:
This is not visible to the users of our library, but there is a reference to `themeContract` in `PreviewCodeSnippet`.